sqlite/src/vdbe.h

```diff
@@ -35,7 +35,9 @@ typedef struct Vdbe Vdbe;
 ** The names of the following types declared in vdbeInt.h are required
 ** for the VdbeOp definition.
 */
+#if !defined(SQLITE_BUILDING_FOR_COMDB2)
 typedef struct sqlite3_value Mem;
```

Comment:
I'm a bit confused about this change.  What was the warning this fixed?

Reply:
double define of typedef (only C11 feature). Mem was defined in one of the includes.